PyRDF/Node.py

```diff
@@ -87,6 +87,11 @@ def _call_handler(self, *args, **kwargs):
         # Handles an operation call to the current node and
         # returns the new node built using the operation call.
 
+        from . import current_backend
+        # Check if the current operation is supported by
+        # the backend
+        current_backend.is_supported(self._cur_attr)
```

Comment:
Can we call it `check_supported`? `is_supported` looks like that it should return a boolean

Reply:
Sure @etejedor . That sounds better 👍